<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPulsedElements</class>
 <widget class="QMainWindow" name="FormPulsedElements">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SR Pulsed</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QELabel" name="qelabel_48">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>$(group):getSequenceCountdown</string>
    </property>
    <property name="variableSubstitutions" stdset="0">
     <string>group=SR-PS-INK</string>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Heating Timer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QESimpleShape" name="qesimpleshape_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>112</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="variable" stdset="0">
     <string>SR-PS-INK:isInterlock</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="titleGroupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>830</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
 	background-color: rgb(255, 255, 255);
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>20</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPID">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>PID Expert Panel </string>
     </property>
    </widget>
   </widget>
   <widget class="QESimpleShape" name="qesimpleshape_22">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>112</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="colour0" stdset="0">
     <color>
      <red>240</red>
      <green>5</green>
      <blue>5</blue>
     </color>
    </property>
    <property name="variable" stdset="0">
     <string>SR-PS-INK:isSequenceReady</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="titleGroupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>831</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
    </property>
    <property name="title">
     <string>Pulsed Injection Power Supplies</string>
    </property>
    <widget class="QESimpleShape" name="qesimpleshape">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):isEnabled</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SR-PS-INS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Septum</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QELabel" name="qelabel_28">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getCurrent</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SR-PS-INS</string>
     </property>
     <property name="notation" stdset="0">
      <enum>QE::Scientific</enum>
     </property>
    </widget>
    <widget class="QELabel" name="qelabel_29">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getVoltage</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SR-PS-INS</string>
     </property>
    </widget>
    <widget class="QENumericEdit" name="txtVoltage">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setVoltage</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SR-PS-INS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="addUnits">
      <bool>false</bool>
     </property>
     <property name="writeOnChange">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QESimpleShape" name="qesimpleshape_2">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>22</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):isTriggerEnabled</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SR-PS-INS</string>
     </property>
    </widget>
    <widget class="QENumericEdit" name="qenumericedit_8">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setVoltage</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SR-PS-INK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="addUnits">
      <bool>false</bool>
     </property>
     <property name="writeOnChange">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QESimpleShape" name="qesimpleshape_3">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>42</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):isTriggerEnabled</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SR-PS-INK</string>
     </property>
    </widget>
    <widget class="QELabel" name="qelabel_30">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getVoltage</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SR-PS-INK</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Kicker</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QELabel" name="qelabel_31">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getCurrent</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SR-PS-INK</string>
     </property>
     <property name="notation" stdset="0">
      <enum>QE::Scientific</enum>
     </property>
    </widget>
    <widget class="QESimpleShape" name="qesimpleshape_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>42</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):isEnabled</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SR-PS-INK</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="qepushbutton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>20</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
     <property name="variable" stdset="0">
      <string>SR-PS-INS:enable</string>
     </property>
     <property name="clickText">
      <string>0</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="qepushbutton_2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>20</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
     <property name="variable" stdset="0">
      <string>SR-PS-INS:enable</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="qepushbutton_5">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>40</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
     <property name="variable" stdset="0">
      <string>SR-PS-INK:enableSequence</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="qepushbutton_6">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>40</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
     <property name="variable" stdset="0">
      <string>SR-PS-INK:enableSequence</string>
     </property>
     <property name="clickText">
      <string>0</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnInjKickerTrigOn">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>40</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
     <property name="variable" stdset="0">
      <string>SR-PS-INK:enableTrigger</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="qepushbutton_8">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>40</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
     <property name="variable" stdset="0">
      <string>SR-PS-INK:disableTrigger</string>
     </property>
     <property name="clickText">
      <string>1</string>
     </property>
    </widget>
    <widget class="QELineEdit" name="qelineedit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setCurrent</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SR-PS-INS</string>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
     <property name="notation" stdset="0">
      <enum>QE::Scientific</enum>
     </property>
    </widget>
    <widget class="QELineEdit" name="qelineedit_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setCurrent</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SR-PS-INK</string>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
     <property name="notation" stdset="0">
      <enum>QE::Scientific</enum>
     </property>
    </widget>
    <widget class="QENumericEdit" name="txtPeakCurrent">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>20</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
     <property name="variable" stdset="0">
      <string>SR-PS-INS:setPeakCurrent</string>
     </property>
     <property name="autoScale">
      <bool>false</bool>
     </property>
     <property name="leadingZeros">
      <number>4</number>
     </property>
     <property name="precision">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
     <property name="writeOnChange">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QELabel" name="qelabel">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>SR-PS-INS:getPeakCurrent</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOFF">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>20</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnON">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>20</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblKick">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string> Countdown</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Trigger</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Interlock</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Kick</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltage</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Peak Current</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSeptumInterlocks">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Details</string>
    </property>
   </widget>
   <widget class="QSimpleShape" name="ledSeptumInterlocks">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="ledType">
     <enum>QSimpleShape::Status</enum>
    </property>
    <property name="colour0" stdset="0">
     <color>
      <red>240</red>
      <green>5</green>
      <blue>5</blue>
     </color>
    </property>
    <property name="colour1" stdset="0">
     <color>
      <red>60</red>
      <green>20</green>
      <blue>20</blue>
     </color>
    </property>
   </widget>
   <widget class="QLabel" name="label_56">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Septum Interlocks</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QELineEdit</class>
   <extends>QLineEdit</extends>
   <header>QELineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QENumericEdit</class>
   <extends>QWidget</extends>
   <header>QENumericEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QEPushButton</class>
   <extends>QPushButton</extends>
   <header>QEPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>QESimpleShape</class>
   <extends>QSimpleShape</extends>
   <header>QESimpleShape.h</header>
  </customwidget>
  <customwidget>
   <class>QSimpleShape</class>
   <extends>QWidget</extends>
   <header>QSimpleShape.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
